jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -209,19 +209,26 @@ YUI.add('user-profile-model-list', function() {
           </div>
         );
       }
-      const list = this.state.modelList;
-      if (!list || list.length === 0) {
-        return null;
-      }
-      const rows = list.map(this._generateRow);
       const props = this.props;
+      const acl = props.acl;
       let createNewButton;
-      if (props.canCreateNew) {
+      if (acl && acl.canAddModels()) {
```

Comment:
The original impl with just acl.canAddModels() seems better to me actually.
It is surely possible that a user has read-only access to the current model but she's also able to create new ones.